supercommand.go

```diff
@@ -16,6 +17,9 @@ import (
 
 var logger = loggo.GetLogger("cmd")
 
+// ErrCommandMissing
+var ErrCommandMissing = stderr.New("command missing")
```

Comment:
This could also be `errors.ConstError("missing command")` since the juju errors package is being imported already anyway. Starting with the verb sounds better as well.

This just appears to be used internally so no need to export. We want to keep the public interface tight.

Reply:
This needs to be exported for the Juju CLI to signal to the supercommand.
See https://github.com/juju/juju/pull/19230 where I've drafted the changes on the CLI side.

Reply:
well.. juju commands will need to return this error, so i think we want it exported.

Reply:
also, naming is hard. this error should not indicate that the command is missing, but rather that it is handing of execution to another command, possibly a plugin.
so perhaps
```
var ErrHandoffExecution = errors.ConstError("handoff execution")
```

Reply:
I'm not seeing ConstError available in `juju/errors`, maybe that something added to Juju 4's error internal error package?
Changed to starting with the verb though.

Reply:
>this error should not indicate that the command is missing, but rather that it is handing of execution to another command, possibly a plugin.

I was considering something like this but I thought it was tying the logic to closely to the perspective of the Juju CLI which will call the plugin. From the perspective of this library, we don't know what the assigned MissingCallback will do.

So I avoided a "handoff execution" style error and just opted for "missing command" since that's sort of all it's doing.